kpi/tests/test_utils.py

```diff
@@ -160,3 +162,18 @@ def test_autovalue_choices_arabic(self):
         part1 = 'العربية'
         part2 = '_001'
         self.assertEqual(surv['choices'][1]['$autovalue'], part1 + part2)
+
+    def test_query_parser(self):
+        query_string = '''
+            (a:a OR b:b AND c:c) AND d:d OR (
+                snakes:🐍🐍 AND NOT alphabet:🍲soup
+            ) NOT 'in a house' NOT "with a mouse"
```

Comment:
`AND` is not required when using `NOT`?

Reply:
Correct! It's assumed to be the same as `AND NOT`